__tests__/app.test.js

```diff
@@ -186,46 +215,74 @@ describe("Error handling", () => {
-    .post ("/api/reviews/6475/comments")
-    .send(comment)
-    .expect(400)
-    .then((response)=>{
-      expect(response.body.msg).toEqual("Invalid id")
-    })
+      .post("/api/reviews/6475/comments")
+      .send(comment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toEqual("Invalid id");
+      });
   });
-  test('POST: 400 ID provided is a string ', () => {
-    const comment = {username: `philippaclaire9`, body: `No comment`}
+  test("POST: 400 ID provided is a string ", () => {
+    const comment = { username: `philippaclaire9`, body: `No comment` };
     return request(app)
-    .post ("/api/reviews/boo/comments")
-    .send(comment)
-    .expect(400)
-    .then((response)=>{
-      expect(response.body.msg).toEqual("Invalid id")
-    })
+      .post("/api/reviews/boo/comments")
+      .send(comment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toEqual("Invalid id");
+      });
   });
-  test('POST: 400 missing body', () => {
-    const comment = {username: `philippaclaire9`}
+  test("POST: 400 missing body", () => {
+    const comment = { username: `philippaclaire9` };
     return request(app)
-    .post ("/api/reviews/2/comments")
-    .send(comment)
-    .expect(400)
-    .then((response)=>{
-      expect(response.body.msg).toEqual("Invalid id")
-    })
+      .post("/api/reviews/2/comments")
+      .send(comment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toEqual("Invalid id");
+      });
   });
-  test('POST: 400 username does not exist', () => {
-    const comment = {username: `someone`, body: `boo`}
+  test("POST: 400 username does not exist", () => {
+    const comment = { username: `someone`, body: `boo` };
     return request(app)
-    .post ("/api/reviews/5/comments")
-    .send(comment)
-    .expect(400)
-    .then((response)=>{
-      expect(response.body.msg).toEqual("Invalid username")
-    })
+      .post("/api/reviews/5/comments")
+      .send(comment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toEqual("Invalid username");
+      });
+  });
+  test("PATCH: 400 wrong id", () => {
+    const obj = {inc_votes:5};
+    return request(app)
+      .patch("/api/reviews/593854")
+      .send(obj)
+      .expect(400)
+      .then((response)=>{
+        expect(response.body.msg).toEqual(`Invalid id`)
+      })
+  });
+  test("PATCH: 400 wrong input type", () => {
+    const obj = {inc_votes:`string`};
+    return request(app)
+      .patch("/api/reviews/5")
+      .send(obj)
+      .expect(400)
+      .then((response)=>{
+        expect(response.body.msg).toEqual(`Invalid data type`)
+      })
+  });
+  test("PATCH: 400 empty object", () => {
+    const obj = {};
+    return request(app)
+      .patch("/api/reviews/5")
+      .send(obj)
+      .expect(400)
+      .then((response)=>{
+        expect(response.body.msg).toEqual(`Invalid data type`)
+      })
```

Comment:
make separate tests for missing username/body to make sure each is being handled